blog/_posts/2019-01-18-fluxdiffeq.md

```diff
@@ -0,0 +1,672 @@
+and [sophisticated event handling](http://docs.juliadiffeq.org/latest/features/callback_functions.html).
+Recently, these native Julia differential equation solvers have successfully been embedded
+into the [Flux.jl](https://github.com/FluxML/Flux.jl/) deep learning package, to allow the use of a full suite of
+highly tested and optimized DiffEq methods within neural networks. Using the new package
+[DiffEqFlux.jl](https://github.com/JuliaDiffEq/DiffEqFlux.jl/),
+we will show the reader how to easily add differential equation
+layers to neural networks using a range of differential equations models, including stiff ordinary
+differential equations, stochastic differential equations, delay differential
+equations, and hybrid (discontinuous) differential equations.
+
+This is the first user-friendly toolbox to combine a fully-featured differential
+equations solver library and neural networks seamlessly together. The blog post will also show
+why the flexibility of a full differential equation solver suite is necessary
+for doing this properly. With the ability to fuse neural networks
+seamlessly with ODEs, SDEs, DAEs, DDEs, stiff equations, and different methods
+for adjoint sensitivity calculations, this is a large generalization of the
+neural ODEs work and will allow researchers to better explore the problem
+domain.
+
+(Note: If you are interested in this work and are an undergraduate or graduate
+student, we have [Google Summer of Code projects available in this area](https://julialang.org/soc). This
+[pays quite well over the summer](https://developers.google.com/open-source/gsoc/help/student-stipends).
+Please join the [Julia Slack](https://slackinvite.julialang.org/) and the #jsoc channel to discuss in more detail.)
+
+## What do differential equations have to do with machine learning?
+
+The first question someone not familiar with the field might ask is, why are
+differential equations important in this context? The simple answer is that a
+differential equation is a way to specify an arbitrary nonlinear transform by
+mathematically encoding prior structural assumptions.
+
+Let's unpack that statement a bit. There are generally three ways
+to define a nonlinear transformation. The first is to explicitly say what it is.
+For example, the sigmoid function is $\sigma(x)=\frac{e^x}{e^x + 1}$. This only works
+if you know the exact functional form that relates the input to the output.
+However, in many cases, such exact relations are not known *a priori*.
+So how do you do nonlinear modeling if you don't know the nonlinearity?
+
+There are two separate ways to then handle this problem. One method is machine
+learning. Usually this is presented as you have some $x$ and you want to predict
+$y$. This generation of a prediction from $x$ is the machine learning model, call it
+$ML$, where you put $x$ into the program and it spits out values $y$. But if you think

```

Comment:
Suggested wording:
```
There are two common ways to handle this problem. One method is to use machine
learning. In a typical machine learning problem, you are given some input $x$ and 
you want to predict an output $y$. This generation of a prediction $y$ from $x$
is a machine learning model (let's call it $ML$).  During training, we attempt to 
adjust the parameters of $ML$ so that it generates accurate predictions.  We
can then use $ML$ for inference (i.e., produce $y$s for novel inputs $x$).
```